### ISSUE TITLE
Show full stack trace in iced stdout, how?

### ISSUE THREAD

**Issue by geekingfrog** [liquidz/vim-iced#414]
Hello. I can't figure out how to get the entire stack trace in the stdout buffer. I only get the last line, which isn't very helpful when it's deep inside some library code.

Here's a quick showcase:

```clojure
(defn boom [x]
  (.substring x 2))

(defn parent-boom [x]
  (boom x))

(comment
  (parent-boom nil))
```

When evaluating the expression in the comment I get:
```
Execution error (NullPointerException) at testing-misc.core/boom (core.clj:13).
Cannot invoke "Object.getClass()" because "target" is null
```
Line `13` is the `.substring` bit. I would like to get the entire stack trace, showing `parent-boom`. For example, if I put a `main` function and run that, I get:

```
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "Object.getClass()" because "target" is null
	at clojure.lang.Reflector.invokeInstanceMethod(Reflector.java:97)
	at testing_misc.core$boom.invokeStatic(core.clj:7)
	at testing_misc.core$boom.invoke(core.clj:6)
	at testing_misc.core$parent_boom.invokeStatic(core.clj:10)
	at testing_misc.core$parent_boom.invoke(core.clj:9)
	at testing_misc.core$_main.invokeStatic(core.clj:15)
	at testing_misc.core$_main.doInvoke(core.clj:12)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at testing_misc.core.main(Unknown Source)
```
Which is what I'd like to see in the stdout buffer. Is there a way to get that?

**Comment by liquidz** [comment]
@geekingfrog Currently there is no way 😢 
Using `printStackTrace` explicitly will show full stack trace to your REPL. (I did if I needed to)
```
(comment
  (try
    (parent-boom nil)
    (catch Exception ex
      (.printStackTrace ex))))
```

But indeed, it is a pain and I was hoping to show full stack trace too.
It should be done by [wrap-stacktrace](https://docs.cider.mx/cider-nrepl/nrepl-api/supplied_middleware.html) middleware in cider-nrepl, so I'll try to do.

**Comment by liquidz** [comment]
OK. I could show full stack trace in stdout buffer for now.
<img width="1137" alt="2022-06-11 7 49 47" src="https://user-images.githubusercontent.com/6941/173160060-a10a1694-cf36-47cd-a01b-8e64a4954a2d.png">

**Comment by liquidz** [comment]
@geekingfrog I've implemented in [dev](https://github.com/liquidz/vim-iced/tree/dev) branch for trial.
When exception occurs, vim-iced will do followings.
* Show full stacktrace to stdout buffer
* Add jumpable files to [quickfix](https://vim-jp.org/vimdoc-en/quickfix.html) 
  * This behavior may help you to jump to the exact line which throws Exception

Could you try dev branch?

**Comment by geekingfrog** [comment]
Everything works as expected now. Populating the quickfix is a nice addition as well <3

**Comment by liquidz** [comment]
@geekingfrog Thanks for your confirmation!
Just released v3.9.2 #416
